app/models/product.rb

```diff
@@ -5,6 +5,8 @@ def validate_each(record, attribute, value)
 end
 
 class Product < ApplicationRecord
+  after_initialize :set_title_default, :set_discount_price_default
```

Comment:
`set_discount_price_default` check whether will this callbacks run for persisted object